pkg/packer/manifests/aws/packer.json.tmpl

```diff
@@ -218,7 +218,7 @@
     },
     {
         "type": "file",
-        "source": "/usr/local/bin/goss",
+        "source": "/usr/local/bin/goss-amd64",
```

Comment:
since the Dockerfile is linking the arch specific binary`ln -s /usr/local/bin/goss-${BUILDARCH} /usr/local/bin/goss`, 
shouldn't we keep it the same path? ` /usr/local/bin/goss`

Reply:
we want to upload the specific amd64 because goss is executed on the remote

Reply:
🤦 . Thank you.